misc/sample_application/README.md

```diff
@@ -0,0 +1,7 @@
+This directory contains the software that is necessary to run a real-time application between a real-time VM and a standard VM using the acrn-hypervisor.

```

Comment:
Can you convert this file into a ReST format (instead of markdown) as this is what we use in this project. Please include a heading too.

Will you provide more extensive documentation in a separate PR?